### PR TITLE
Feature/th new fetch methods

### PR DESCRIPTION
**Pull request by relyt4me**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description / What does this PR do?
Three new methods for getting a users rating posting a new rating and also deleting an existing rating

### Files modified:
Fetch

### Has This Been Tested?
- [ ] No
- [x] Yes

### Are there specific methods for testing?
Postman testing for all fetches

### Message/Questions for reviewer:
The error catches for this are set up but may need to be changed based on what we want to use the errors for.

### Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Fetch.js [hunk 1]
```diff
@@ -15,6 +15,49 @@ class Fetch {
...
+      .then((data) => {
+        return data;
+      })
+      .catch((err) => {
+        //We may want to refactor this if after we find out where we want to use these errors if we want to use them
+        return err;
+      });
+  }
+
+  deleteRatingForUser(userId, ratingId) {
```
Comment (NicoleLatifi) [inline src/Fetch.js:49]: Have you tested this delete request? I haven't created a delete before so I'm excited to see it in action.

File: src/Fetch.js [hunk 2]
```diff
@@ -15,6 +15,49 @@ class Fetch {
...
+      .then((data) => {
+        return data.ratings;
+      })
+      .catch((err) => {
+        return err;
+      });
+  }
+
+  addRatingForUser(userId, movieId, ratingInt) {
+    // In this fashion we would need to fetch new ratings after we add the Ratings, unsure as of yet
```
Comment (NicoleLatifi) [inline src/Fetch.js:30]: That's how I've done a post before, posted and then did a get.

File: src/Fetch.js (hunk 1 above)
Comment (relyt4me) [reply src/Fetch.js:49]: I tested in Postman, and used the format I've used in previous projects. I believe it should function fine.

File: src/Fetch.js (hunk 2 above)
Comment (relyt4me) [reply src/Fetch.js:30]: Figured, but perhaps we can handle it with a state change and have react do it for us.

Comment (NicoleLatifi) [reply src/Fetch.js:30]: I think if that works, it would be a better approach.

**Pull request merged**